modules/renter/contractor/formcontract.go

```diff
@@ -113,7 +114,8 @@ func (c *Contractor) managedNewContract(host modules.HostDBEntry, numSectors uin
 
 	contractValue := contract.RenterFunds()
 	c.log.Printf("Formed contract with %v for %v SC", host.NetAddress, contractValue.Div(types.SiacoinPrecision))
-
+	contract.MerkleRoots = []crypto.Hash{}
+	c.cachedRevisions[contract.ID] = cachedRevision{contract.LastRevision, contract.MerkleRoots}
```

Comment:
oh, this was a temporary measure that the old (generic) journal required. I can remove it now